moztelemetry/stats.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from __future__ import division
+
+import math
+from collections import namedtuple
+
+
+def rank(sample):
+    rank = 1
+    ranks = {}
+
+    for k in sorted(sample.keys()):
+        n = sample[k]
+        ranks[k] = rank + (n - 1) / 2
+        rank += n
+
+    return ranks
+
+
+def tie_correct(sample):
+    tc = 0
+    n = sum(sample.values())
+
+    if n == 1:
+        return 1.0  # Avoid a ``ZeroDivisionError``.
+
+    for k in sorted(sample.keys()):
+        tc += math.pow(sample[k], 3) - sample[k]
+    tc = 1 - tc / (math.pow(n, 3) - n)
+
+    return tc
+
+
+def ndtr(v):
```

Comment:
`ndtr` takes the integral from `-infinity` to `v` over the gaussian distribution. This is why the input below is `-abs(z)`. Can we do two things here:

- [ ] Add tests that this function returns the same value as [scipy ndtr](https://docs.scipy.org/doc/scipy/reference/generated/scipy.special.ndtr.html#scipy.special.ndtr)

- [ ] Document what this function is doing, with a reference to the above link

I do like calculating this separately, rather than importing scipy.